doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -0,0 +1,90 @@
+.. _thread_ot_memory:
+
+OpenThread memory requirements
+##############################
+
+This page provides information about the layout and the amount of Flash and RAM memory required by the sample using the OpenThread stack.
+These values are useful to see whether your application has enough space for running on paricular platforms in different OpenThread configurations.
+
+
+Flash and RAM requirements for nRF52840 and nRF52833
+****************************************************
+
+The following tables list flash memory and RAM requirements for the provided samples built using the OpenThread libraries.
+The values change depending on the application and hardware platform.
```

Comment:
Is this valid for all samples for Thread in NCS?
By `OpenThread libraries`, you mean the pre-built libraries?

Reply:
Yes, yes.